paper-server/patches/sources/net/minecraft/server/commands/GiveCommand.java.patch

```diff
@@ -8,6 +8,28 @@
          int maxStackSize = itemStack.getMaxStackSize();
          int i = maxStackSize * 100;
          if (count > i) {
+@@ -66,7 +_,9 @@
+                     ItemStack itemStack1 = item.createItemStack(min, false);
+                     boolean flag = serverPlayer.getInventory().add(itemStack1);
+                     if (flag && itemStack1.isEmpty()) {
+-                        ItemEntity itemEntity = serverPlayer.drop(itemStack, false);
++                        // Paper start - Fix PlayerDropItemEvent dupe on /give
++                        ItemEntity itemEntity = serverPlayer.drop(itemStack, false, false, false, null);
++                        // Paper end - Fix PlayerDropItemEvent dupe on /give
```

Comment:
Paper start/end comments aren't necessary for a one line change, you can just put a comment at the end of the line. I think the comment itself should also be changed to better describe the change, something along the lines of not firing the drop item event for the give command.